### PR TITLE
Temperature dependent latent heat

### PR DESCRIPTION
**Pull request by nbren12**
[Implement phase dependent condensation updates](https://github.com/ai2cm/fv3net/commit/ba4b4febdcc38aed9f14a4fb57e9df7733e75ae6). This could be relevant to online cloud and temperature drifts.

Coverage reports (updated automatically):
- test_unit: [82%](https:\/\/output.circle-artifacts.com\/output\/job\/439fb4f6-578c-4d0b-acd5-525fea7854d1\/artifacts\/0\/tmp\/coverage\/htmlcov-test_unit\/index.html)

### CODE REVIEW

File: external/emulation/emulation/zhao_carr.py [hunk 1]
```diff
@@ -68,6 +82,49 @@ def squash_precpd(state, emulator, cloud_squash):
 def apply_condensation_liquid_phase(state, net_condensation):
     # from physcons.f
     lv = 2.5e6
+    return apply_condensation(state, net_condensation, lv=lv)
+
+
+@numba.njit
+def ice_water_number(temperature_celsius, cloud):
```
Comment (oliverwm1) [inline external/emulation/emulation/zhao_carr.py:89]: This is the fraction of condensate that is ice? The function name confused me... I was expecting a number concentration.

Comment (oliverwm1) [reply external/emulation/emulation/zhao_carr.py:89]: I guess it's more a flag than a fraction.

**Review by oliverwm1 — APPROVED** [review]
LGTM. The `ice_water_number` function name confused me for a while... the Fortran calls it something more like `ice_water_id_number`.

File: external/emulation/emulation/zhao_carr.py (hunk 1 above)
Comment (nbren12) [reply external/emulation/emulation/zhao_carr.py:89]: I renamed it to ice_water_flag.

**Pull request merged**